Packages/Tracking/Hands/Runtime/Scripts/HandModelManager.cs

```diff
@@ -0,0 +1,512 @@
+                {
+                    if (handModel.Handedness == Chirality.Left)
+                    {
+                        handModelPairs[i].Left = null;
+                    }
+                    else
+                    {
+                        handModelPairs[i].Right = null;
+                    }
+
+                    if (handModelPairs[i].Left == null && handModelPairs[i].Right == null)
+                    {
+                        handModelPairs.RemoveAt(i);
+                    }
+                    return;
+                }
+            }
+        }
+
+        #region Getters
+
+        /// <summary>
+        /// Returns all hands which will be active if a hand appears in the scene
+        /// This includes hands whose EnableDisable is not frozen, or are frozen in an active state
+        /// </summary>
+        /// <returns>Returns all hands which will be active if a hand appears in the scene</returns>
+        public List<HandModelBase> GetActiveHandsInScene()
+        {
+            List<HandModelBase> activeHands = new List<HandModelBase>();
+            foreach (HandModelPair handModelPair in handModelPairs)
+            {
+                if (!handModelPair.LeftEnableDisable.FreezeHandState ||
+                    handModelPair.LeftEnableDisable.FreezeHandState && handModelPair.Left.isActiveAndEnabled)
+                {
+                    activeHands.Add(handModelPair.Left);
+                }
+
+                if (!handModelPair.RightEnableDisable.FreezeHandState ||
+                    handModelPair.RightEnableDisable.FreezeHandState && handModelPair.Right.isActiveAndEnabled)
+                {
+                    activeHands.Add(handModelPair.Right);
+                }
+            }
+            return activeHands;
+        }
+
+        /// <summary>
+        /// Returns a hand model pair by index, if given a valid index
+        /// </summary>
+        /// <param name="index">The index of the hand model pair</param>
+        /// <param name="handModelPair">The hand model pair to be returned</param>
+        /// <returns>Returns true if the pair have been found, and false if not</returns>
+        public bool GetHandModelPair(int index, out HandModelPair handModelPair)
+        {
+            handModelPair = null;
+            if (index >= 0 && index < handModelPairs.Count)
+            {
+                handModelPair = handModelPairs[index];
+                return true;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Returns a hand model pair by index, if given a valid index
+        /// </summary>
+        /// <param name="handModelPairID">The handModelPairID of the hand model pair</param>
+        /// <param name="handModelPair">The hand model pair to be returned</param>
+        /// <returns>Returns true if the pair have been found, and false if not</returns>
+        public bool GetHandModelPair(string handModelPairID, out HandModelPair handModelPair)
```

Comment:
Would be better named TryGetHandModelPair as it matches the signature of other .NET methods,

Reply:
✔️